photoeditor/src/main/java/ja/burhanrashid52/photoeditor/PhotoEditorImpl.java

```diff
@@ -349,4 +346,36 @@ public void setOnPhotoEditorListener(@NonNull OnPhotoEditorListener onPhotoEdito
     public boolean isCacheEmpty() {
         return viewState.getAddedViewsCount() == 0 && viewState.getRedoViewsCount() == 0;
     }
+
+    // region Shape
+    @Override
+    public void addShape(Shape shape) {
```

Comment:
We can directly take a `Shape` or `ShapeBuilder`

Reply:
OK